src/main/drivers/pwm_output.c

```diff
@@ -38,7 +38,7 @@ static pwmOutputPort_t servos[MAX_SUPPORTED_SERVOS];
 
 bool pwmMotorsEnabled = false;
 
-static void pwmOCConfig(TIM_TypeDef *tim, uint8_t channel, uint16_t value, uint8_t output)
+static void pwmOCConfig(TIM_TypeDef *tim, uint8_t channel, uint16_t value, uint8_t output, uint8_t polarity)
```

Comment:
Yeah, you're right, I completely forgot that I was using `TIMER_OUTPUT_INVERTED` for that matter. I just have to invert this bit. Thanks.